zeebe/bpmn-model/src/test/java/io/camunda/zeebe/model/bpmn/validation/ZeebeTaskListenersValidationTest.java

```diff
@@ -54,9 +62,43 @@ void testEventTypeNotDefined() {
             ReflectUtil.getResourceAsStream(
                 "io/camunda/zeebe/model/bpmn/validation/ZeebeTaskListenersValidationTest.testEventTypeNotDefined.bpmn"));
 
+    // when/then
+    // as we don't fully support TL yet, two violations are expected here.
+    ProcessValidationUtil.assertThatProcessHasViolations(
+        process,
+        expect(ZeebeTaskListener.class, "Attribute 'eventType' must be present and not empty"),
+        expect(
+            ZeebeTaskListener.class,
+            "Task listeners are not yet supported. Java BPMN-modeling API for task"
+                + " listeners was introduced with version 8.6, but the support for listener execution will "
+                + "be added in the upcoming versions."));
+  }
+
+  @Test
+  @DisplayName("task listener with unsupported `eventType` property")
+  void testEventTypesNotSupported() {
```

Comment:
🔧 We can use `@EnumSource` to parametrize this test to execute this test for each task listener event type:
```suggestion
  @DisplayName("task listener with unsupported `eventType` property")
  @ParameterizedTest(name = "unsupported event type: ''{0}''")
  @EnumSource(ZeebeTaskListenerEventType.class)
  void testEventTypesNotSupported(final ZeebeTaskListenerEventType unsupportedEventType) {
```